app/controllers/home_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  def index
+    if user_signed_in?
+      redirect_to books_path
+    else
+      redirect_to new_user_session_path
+    end
```

Comment:
```suggestion
    redirect_to books_path
```

とすれば、未ログイン状態でbooks_pathにリダイレクトしても、強制的にログイン画面にリダイレクトされると思うのですがいかがでしょうか？

Reply:
確かに。。。その通りです。ここだけ丁寧に分岐してもあまり意味がないと思います。